docs/source-pytorch/fabric/api/fabric_args.rst

```diff
@@ -35,7 +35,7 @@ The ``"auto"`` option recognizes the machine you are on and selects the availabl
 strategy
 ========
 
-Choose a training strategy: ``"dp"``, ``"ddp"``, ``"ddp_spawn"``, ``"tpu_spawn"``, ``"deepspeed"``, ``"ddp_sharded"``, or ``"ddp_sharded_spawn"``.
+Choose a training strategy: ``"dp"``, ``"ddp"``, ``"ddp_spawn"``, ``"xla"``, ``"deepspeed"``, ``"fsdp"````.
```

Comment:
is `xla` now valid with `tpu` and `gpu`? Also with single instances of each accelerator or only multi-device?

Reply:
> is xla now valid with tpu and gpu?

No. Needs #16130 

> Also with single instances of each accelerator or only multi-device?

Multi-device, otherwise it is `strategy='single_tpu'`.

Reply:
The name xla was chosen here to be future proof :))
200 IQ move from us